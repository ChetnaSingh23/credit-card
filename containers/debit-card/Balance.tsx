import { View, Text } from '../../components/Themed';
import { styles } from  './debit.style';

export const Balance = () => {
  return (
    <View style={styles.balanceWrap}>
      <View style={styles.cashView}>
          <Text style={styles.cashViewText}>S$</Text>
      </View>
      <Text style={[styles.cashViewText, styles.balance]}>3,000</Text>
    </View>
  );
}
