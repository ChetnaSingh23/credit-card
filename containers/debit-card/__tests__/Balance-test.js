import * as React from 'react';
import renderer from 'react-test-renderer';

import {Balance} from '../Balance';

it(`Balance component renders correctly`, () => {
  const tree = renderer.create(<Balance></Balance>).toJSON();

  expect(tree).toMatchSnapshot();
});
