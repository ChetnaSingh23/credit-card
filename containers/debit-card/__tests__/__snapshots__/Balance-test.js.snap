// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Balance component renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgb(12, 54, 90)",
      },
      Object {
        "display": "flex",
        "flexDirection": "row",
        "marginTop": 20,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "rgb(12, 54, 90)",
        },
        Object {
          "alignItems": "center",
          "backgroundColor": "#01D167",
          "borderRadius": 5,
          "color": "#FFFFFF",
          "display": "flex",
          "height": 30,
          "justifyContent": "center",
          "width": 50,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#000",
          },
          Object {
            "fontSize": 20,
            "fontWeight": "bold",
          },
        ]
      }
    >
      S$
    </Text>
  </View>
  <Text
    style={
      Array [
        Object {
          "color": "#000",
        },
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "bold",
          },
          Object {
            "marginLeft": 15,
          },
        ],
      ]
    }
  >
    3,000
  </Text>
</View>
`;
